### PR TITLE
Add include_values_in_response to append_row & append_rows

### PR DESCRIPTION
**Pull request by martimarkov**

### CODE REVIEW

**Comment by martimarkov** [comment]
Can we merge this?

**Comment by punnerud** [comment]
There is several parameters that the API-support. Would it be better to add **kwargs so include_values_in_response and others can be supported?

Something like this:
```
append_row(
        value_input_option='RAW',
        insert_data_option=None,
        table_range=None,
        , **kwargs
    ):

if kwargs is not None:
            payload = {**payload, **kwargs}
```

File: gspread/models.py [hunk 1]
```diff
@@ -1500,6 +1509,7 @@ def append_rows(
         params = {
             'valueInputOption': value_input_option,
             'insertDataOption': insert_data_option,
+            'includeValuesInResponse': include_values_in_response,
```
Comment (martimarkov) [inline gspread/models.py:1512]: @punnerud But in here we still would need custom logic as for example `value_input_option` is not used here. I don't remember the API specs on the top of my head but maybe there are other things like that?

Comment (punnerud) [reply gspread/models.py:1512]: Yes, we need to keep every parameter that have logic. All others could be passed directly to the API, and let the API give errors if they don’t exists.

Should make it a bit better than my previous code example, so the parameters can be placed in random order.

**Comment by lavigne958** [comment]
closing this PR in favor of #957 .

@martimarkov  I cherry-picked your commit, this way you keep your contribution.

**Pull request closed without merging**